controllers/topics-controllers.js

```diff
@@ -0,0 +1,15 @@
+const { selectTopics } = require("../models/topics-models");
+
+exports.getTopics = (request, response, next) => {
+  const order = request.query.order || "asc";
+  const sort = request.query.sort_by || "slug";
+  const slug = request.query.slug;
```

Comment:
I like the consideration of queries - but this is way too early in the sprint for it - we want to laser focus on what's asked for on in the task so please resubmit without the queries. They come into play later, so feel free to save this code in another file